.gitignore

```diff
@@ -1,6 +1,7 @@
 # Dependency directory
 node_modules
 lib/
+dist/
```

Comment:
FYI - Looks like `dist/` isn't included in the latest package after this was released:
```
ls node_modules/\@google/semantic-release-replace-plugin/
LICENSE      README.md    package.json
```

Reply:
`@google/semantic-release-replace-plugin` is fixed in 1.2.5 and `semantic-release-replace-plugin` is fixed in 1.2.4